.isort.cfg

```diff
@@ -1,2 +1,2 @@
 [settings]
-known_third_party = asv,click,clickhouse_driver,dateutil,google,graphviz,impala,jinja2,kudu,multipledispatch,numpy,pandas,pkg_resources,plumbum,psycopg2,pyarrow,pydata_google_auth,pymapd,pymysql,pytest,pytz,regex,requests,ruamel,setuptools,sphinx_rtd_theme,sqlalchemy,toolz
+known_third_party = asv,click,clickhouse_driver,dateutil,geopandas,google,graphviz,impala,jinja2,kudu,multipledispatch,numpy,pandas,pkg_resources,plumbum,psycopg2,pyarrow,pydata_google_auth,pymapd,pymysql,pytest,pytz,regex,requests,ruamel,setuptools,sphinx_rtd_theme,sqlalchemy,toolz
```

Comment:
maybe also gdal and geoalchemy2 should be added here.

Reply:
This was automatically added by the pre-commit hook. Should I edit it manually?

Reply:
actually not sure .. @cpcloud do you know?

Reply:
No, this is fine. Third-party dependencies get automatically added when they are detected.

Reply:
We should only add gdal and geoalchemy2 if they are imported directly inside of ibis *and* they are not detected by `seed-isort-config`.

Reply:
`geoalchemy2` *is* imported below, and it seems like it has not been picked up by `isort`. `gdal` is not directly imported. Should I add `geoalchemy2`?

Reply:
Yes, please do.

Reply:
Hmm, `seed-isort-config` doesn't seem to like my manually adding imports here. It is automatically removing them in the pre-commit hook.

Reply:
Ok, I think that's probably fine. Does it cause issues elsewhere? If not then I'd say leave as is.

Reply:
Doesn't seem to cause any issues.

Reply:
And now the file is unchanged. It seems that `isort` isn't smart enough to see `pytest.importorskip` or imports in a `try`/`except` block.